snap/info.go

```diff
@@ -65,8 +72,13 @@ type SlotInfo struct {
 type AppInfo struct {
 	Snap *Info
 
-	Name    string
-	Command string
-	Plugs   map[string]*PlugInfo
-	Slots   map[string]*SlotInfo
+	Name        string
+	Command     string
+	Description string
```

Comment:
Sergio Schvezov, [05.04.16 20:52]
@niemeyer @pedronis during Gramado we killled description for apps; it was originally intended for services (systemd) but it makes little sense as it is not user facing and if it comes to it we can generate one (if the unit name itself doesn't help already)